wavefront/values.yaml

```diff
@@ -209,6 +209,10 @@ proxy:
   ## This is usually 30001
   # jaegerPort: 30001
 
+  ## TCP ports to receive Jaeger Thrift formatted data via HTTP. The data is then sent to Wavefront in Wavefront span format.
+  ## Comma-separated list of available port numbers. Can be a single port.
```

Comment:
We should probably support only a single port via helm similar to the other ones. Let's rename this property to `traceJaegerHttpListenerPort` accordingly